src/Console/Command/ProcessCommand.php

```diff
@@ -88,6 +95,7 @@ public function __construct(
         $this->afterRectorCodingStyle = $afterRectorCodingStyle;
         $this->configuration = $configuration;
         $this->rectorApplication = $rectorApplication;
+        $this->fileExtensions = $parameterProvider->provideParameter('file_extensions') ?? [];
```

Comment:
Acutally, this can be autowired as `$fileExtensions` parameter.

```php
/**
 * @param string[] $fileExtensions
 */
public function __construct(array $fileExtensions)
{
    $this->fileExtensions = $fileExtensions;
}
```